src/unsubscriber.py

```diff
@@ -1,9 +1,10 @@
-import re
-from typing import Optional
-import boto3
-import os
 import logging
-import cfnresponse
+import os
+import re
+from typing import Optional, TypedDict
+
+import boto3  # type: ignore
+import cfnresponse  # type: ignore
```

Comment:
I tried to get rid of cfnreponse because it won't work with terraform invocation.

Reply:
I think it would be best to get rid of it tbh, or only import it conditionally. Alternatively, we can import the code into this code base https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-lambda-function-code-cfnresponsemodule.html#cfn-lambda-function-code-cfnresponsemodule-source-python which I think we should do anyway as otherwise linters/checkers fail for it.

We then call it conditionally.

Reply:
Pushed